ggplot/geoms/geom.py

```diff
@@ -199,7 +209,7 @@ def _find_aes_and_data(self, args, kwargs):
 
         for arg in args:
             if isinstance(arg, aes) and passed_aes:
-                raise Execption(aes_err)
+                raise Exception(aes_err)
```

Comment:
There are still a couple of those lying around and some have been fixed in my current branch. I plan to grep for all errors and make sure they are the right type.